lc2fen/board2data.py

```diff
@@ -12,21 +15,20 @@
 from tqdm import tqdm
 
 from lc2fen.detectboard import detect_board
-from lc2fen.fen import fen_to_board, rotate_board_fen2image
-from lc2fen.split_board import split_square_board_image, split_board_image
+from lc2fen.fen import fen_to_board, rotate_board_from_standard_view
+from lc2fen.split_board import split_board_image_trivial, split_board_image_advanced
 
 
-def regenerate_data_state(data_path):
-    """
-    Regenerates the state of the data directory.
+def regenerate_data_folder(data_path: str):
+    """Regenerate the data folder.
 
-    Deletes all files and subdirectories in 'data_path/boards/output',
-    'data_path/boards/debug_steps' and 'data_path/pieces'. Regenerates
-    these directories and the pieces subdirectories.
+    This function deletes the "`data_path`/boards/output" folder, the
+    "`data_path`/boards/debug_steps" folder, and the "`data_path`/pieces"
+    folder, and then it regenerates these folders (the folders will be
+    empty) and creates the 13 piece subfolders.
 
-    :param data_path: Path to the 'data' folder. For example: '../data'.
+    :param data_path: Path to the data folder (for example, `"data"`).
```

Comment:
Good observation, but I was thinking that if I could fit the example(s) in one line (same line as the parameter summary), I should do it (as it would show up in the function preview, which could be convenient). It's not always possible to fit in one line, and that's when I put the example(s) in a separate line:
![image](https://github.com/davidmallasen/LiveChess2FEN/assets/51491203/05210be4-19fe-4d46-85ae-ae2bfde49574)

But I also noticed that I used the "Example:" format instead of the "(for example, )" format in cases where it's possible to fit in one line, so I'll change those "(for example, )" instances to the "Example: " instances:
![image](https://github.com/davidmallasen/LiveChess2FEN/assets/51491203/9907e812-fc8a-4601-8246-30886e1123f6)